packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/User.kt

```diff
@@ -68,6 +68,34 @@ public interface User {
      */
     public val identities: List<UserIdentity>
 
+    /**
+     * Returns the provider type used to log the user
+     *
+     * @return the provider type of the user
```

Comment:
We don't really use `@return` for properties. I would just remove this since the above sentence covers everything.